### PR TITLE
added changes for 1.9 upgrade

### PR DESCRIPTION
**Pull request by dereckmartin**
Updated changes to support 1.9 upgrade

### CODE REVIEW

File: templates/harbor.yml.epp [hunk 1]
```diff
@@ -112,6 +113,11 @@ jobservice:
   # Maximum number of job workers in job service  
   max_job_workers: <%= $max_job_workers %>
 
+<% if ersioncmp($cfg_version, '1.9.0') >= 0 { -%>
+notification:
+  webhook_job_max_rety: <%= $webhook_job_max_rety %>
```
Comment (walkamongus) [inline templates/harbor.yml.epp]: typo?

File: templates/harbor.yml.epp [hunk 2]
```diff
@@ -120,6 +126,18 @@ chart:
 log:
   # options are debug, info, warning, error, fatal
   level: <%= $log_level %>
+
+<% if ersioncmp($cfg_version, '1.9.0') >= 0 { -%>
```
Comment (walkamongus) [inline templates/harbor.yml.epp]: missing v?

File: templates/harbor.yml.epp [hunk 3]
```diff
@@ -112,6 +113,11 @@ jobservice:
   # Maximum number of job workers in job service  
   max_job_workers: <%= $max_job_workers %>
 
+<% if ersioncmp($cfg_version, '1.9.0') >= 0 { -%>
```
Comment (walkamongus) [inline templates/harbor.yml.epp]: missing v?

Comment (dereckmartin) [reply templates/harbor.yml.epp]: is $cfg_version not the same as the on in config.pp

File: templates/harbor.yml.epp (hunk 2 above)
Comment (dereckmartin) [reply templates/harbor.yml.epp]: is $cfg_version not the same as the on in config.pp

Comment (walkamongus) [reply templates/harbor.yml.epp]: I meant versioncmp is missing the leading character -- `ersioncmp`

File: templates/harbor.yml.epp (hunk 1 above)
Comment (walkamongus) [reply templates/harbor.yml.epp]: looks like it should be `webhook_job_max_retry`

**Pull request merged**